src/ImageRunModal.jsx

```diff
@@ -386,27 +380,33 @@ export class ImageRunModal extends React.Component {
             body: "",
         }));
 
-        // If there are registries configured search in them, or if a user searches for `docker.io/cockpit` let
-        // podman search in the user specified registry.
-        if (Object.keys(this.props.podmanInfo.registries).length !== 0 || value.includes('/')) {
-            searches.push(this.activeConnection.call({
-                method: "GET",
-                path: client.VERSION + "libpod/images/search",
-                body: "",
-                params: {
-                    term: value,
-                }
-            }));
-        } else {
-            searches = searches.concat(utils.fallbackRegistries.map(registry =>
-                this.activeConnection.call({
+        // Don't start search queries when tag is specified as search API doesn't support
+        // searching for specific image tags
+        // instead only rely on manifests query (requires full image url)
+        const patt = /:[\w-|\d]+$/;
+        if (!patt.test(value)) {
```

Comment:
Either define `patt` at the top level as `RE_TAG` or so and recycle it  (preferable IMHO), or write it shorthand as

```js
if (!patt.matches(/:[\w-\d]+$/))
```